main/lsp/requests/code_action.cc

```diff
@@ -20,8 +20,6 @@ unique_ptr<ResponseMessage> CodeActionTask::runRequest(LSPTypecheckerDelegate &t
 
     vector<unique_ptr<CodeAction>> result;
 
-    prodCategoryCounterInc("lsp.messages.processed", "textDocument.codeAction");
```

Comment:
Moved to a few centralized locations, since LSPTasks now have a methodString that can be used to get the category.